aiida_pseudo/cli/install.py

```diff
@@ -108,9 +110,27 @@ def download_sssp(
     from aiida_pseudo.groups.family import SsspFamily
     from .utils import attempt
 
-    url_sssp_base = 'https://legacy-archive.materialscloud.org/file/2018.0001/v4/'
-    url_archive = f"{url_sssp_base}/{SsspFamily.format_configuration_filename(configuration, 'tar.gz')}"
-    url_metadata = f"{url_sssp_base}/{SsspFamily.format_configuration_filename(configuration, 'json')}"
+    url_prefix = 'https://staging-invenio.materialscloud.org/record/file?filename='
+    url_suffix = '&parent_id=19'
+
+    url_version_yaml = f"{url_prefix}versions.yaml{url_suffix}"
+
+    with attempt('downloading patch versions information... ', include_traceback=traceback):
+        response = requests.get(url_version_yaml)
+        response.raise_for_status()
+        version_mapping = yaml.load(response.content, Loader=yaml.FullLoader)
+
+    echo.echo_info(f'Latest patch version found: {version_mapping[configuration.version]}')
+
+    archive_filename = SsspFamily.format_configuration_filename(
+        configuration, 'tar.gz', version_mapping[configuration.version]
```

Comment:
Why did you add the argument `patch_version` to `format_configuration_filename` if you are not using it here. What will the output of `version_mapping[configuration.version]` be? Does that contain the patch version? Think it would be good to add a comment above line 121 with an example of the structure of the version mapping yaml returned by the first request.

Reply:
I am using it, no? `version_mapping[configuration.version]` will indeed return the patch version, but probably best to just grab this once and put it in a `patch_version` variable.

Reply:
I see. Maybe would be useful to use the keyword to make it clearer. The fact that it also isn't very clear, is because it is not immediately clear what the `version_mapping` contains. If you add a comment there, it makes everything easier to comprehend